k8s.go

```diff
@@ -258,6 +258,10 @@ type KubernetesNodePoolProperties struct {
 	// The desired Maintanance Window
 	// Required: false
 	MaintenanceWindow *MaintenanceWindow `json:"maintenanceWindow,omitempty"`
+
+	// Public Ips
+	// Required: false
+	PublicIps	[]string	`json:"publicIps,omitempty"`
```

Comment:
- Golang conventions would have required `PublicIPs` here. It would also have been consistent with `KubernetesNodeProperties`'s `PublicIP` field.
- The unit tests haven't been adapted.
- `gofmt` or `goimports` would have removed lots of spaces in this line. The lack of CI has hit us again.

Maybe some MRs get merged a bit too fast. If you want us to review some changes, let us know or tag us or assign us. :)